test-lib.sh

```diff
@@ -1310,6 +1310,11 @@ ct_run_tests_from_testset() {
         TESTSUITE_RESULT=1
       fi
     fi
+    # As soon as test is finished
+    # switch the project from sclorg-test-<NUMBER> to default.
+    if [ "${CT_OCP4_TEST:-false}" == "true" ]; then
```

Comment:
Handling OC stuff does not really seem to fit to this function.  It is not related to executing tests.
However, I understand, that adding this line to all of the containers is not really great solution too.

What about adding these lines to the openshift cleanup function `ct_os_cleanup`?

Reply:
This function has to be called after each test and not after the whole test suite.

Reply:
Oh, I see now, thanks for explanation. I am OK with this solution then.